launcher/src/main/java/com/lightstep/opentelemetry/launcher/OpenTelemetryConfiguration.java

```diff
@@ -25,15 +27,16 @@
     private Propagator propagator;
 
     private static final Map<Propagator, HttpTextFormat> PROPAGATORS =
-        ImmutableMap.of(
-            Propagator.TRACE_CONTEXT,
-            new HttpTraceContext(),
-            Propagator.B3,
-            B3Propagator.getMultipleHeaderPropagator(),
-            Propagator.B3_SINGLE,
-            B3Propagator.getSingleHeaderPropagator(),
-            Propagator.JAEGER,
-            new JaegerPropagator());
+        new HashMap<Propagator, HttpTextFormat>() {
```

Comment:
Makes me wonder about having a `Builder` at the SDK level to automatically configure this, just like we do for the `SpanProcessor`s there ;)